client/components/App.jsx

```diff
@@ -9,9 +9,9 @@ export default class App extends React.Component {
   render () {
     return (
       <div className='app'>
-        <MessageList messages={this.state.messageData} />
+        {/* <MessageList messages={this.state.messageData} /> */}
```

Comment:
You could just show them both and then have whoever does the react-router task to sort it out.